src/lib.rs

```diff
@@ -15,6 +15,7 @@
 #![feature(conservative_impl_trait)]
 #![feature(generator_trait)]
 #![feature(use_extern_macros)]
+#![feature(never_type)]
```

Comment:
I'd personally prefer to not block on this stabilizing, could we avoid this feature?

Reply:
Switched to a custom uninhabited type.